apps/frontend/src/utilities/export_util.ts

```diff
@@ -1,3 +1,39 @@
+import concat from 'concat-stream';
+import {saveAs} from 'file-saver';
+import {ZipFile} from 'yazl';
+
+type File = {
+  filename: string;
+  data: string;
+};
+export async function saveSingleOrMultipleFiles(
+  files: File[],
+  filetype: string
+) {
+  if (files.length === 1) {
+    const blob = new Blob([files[0].data], {
+      type: 'application/csv'
```

Comment:
The filetype when called from the ExportJson.vue (line 55) is json - shouldn't the type here be set accordantly?